### PR TITLE
Split PR profiling job with multiple workflows into multiple jobs one per workflow.

### PR DESCRIPTION
**Pull request by gartung**
The profiling job is timing out for some pull requests during the high pileup workflow. 
This should allow other workflow profiles to be reported.

### CODE REVIEW

**Comment by gartung** [comment]
enable profiling

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
abort

**Comment by gartung** [comment]
please test with https://github.com/cms-sw/cmssw/pull/38147

**Comment by gartung** [comment]
enable profiling

**Comment by gartung** [comment]
please test

File: pr_testing/test_multiple_prs.sh [hunk 1]
```diff
@@ -1029,7 +1029,9 @@ if [ "X$BUILD_OK" = Xtrue -a "$RUN_TESTS" = "true" ]; then
   if [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep '^PROFILING$' | wc -l) -gt 0 ] ; then
     if $PRODUCTION_RELEASE ; then
       DO_PROFILING=true
-      mark_commit_status_all_prs 'profiling' 'pending' -u "${BUILD_URL}" -d "Waiting for tests to start"
+      for wf in $PROFILING_WORKFLOWS;do
```
Comment (smuzaffar) [inline pr_testing/test_multiple_prs.sh:1032]: @gartung , `PROFILINH_WORKFLOWS` are not define here. you need to get these from `$CMS_BOT_DIR/cmssw-pr-test-config` just like https://github.com/cms-sw/cms-bot/blob/master/pr_testing/run-pr-profiling.sh#L4

Comment (gartung) [reply pr_testing/test_multiple_prs.sh:1033]: Thanks for spotting that. I was wondering why no profiling jobs started.

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
please test

**Comment by smuzaffar** [comment]
+externals
looks good

**Pull request merged**